### PR TITLE
Bug 1123266 - Connect Settings and Tutorial r=crdlc

### PR DESCRIPTION
**Pull request by borjasalguero**

### CODE REVIEW

File: app/js/screens/settings.js [hunk 1]
```diff
@@ -114,8 +115,23 @@
...
         // Add listeners just once
+        _tutorialButton.addEventListener(
+          'click',
+          function() {
+            Loader.getWizard().then(function(Wizard) {
+              Wizard.init(
+                true /*Boot from the first slide*/,
+                true /*Is a tutorial, no auth. is needed*/
+              );
+              Settings.hide();
```
Comment (crdlc) [inline app/js/screens/settings.js]: Maybe we could hide settings and once finished this transition we could go to wizard, what do you think?

````js
Settings.hide(Navigation.to.bind(null, 'wizard-panel'));
```

File: app/js/screens/wizard/authenticate.js [hunk 2]
```diff
@@ -24,22 +25,24 @@
...
-        _fxaButton = document.getElementById('authenticate-fxa-button');
-      }
-
-      if (!_mobileIdButton) {
-        _mobileIdButton = document.getElementById('authenticate-msisdn-button');
-      }
-
-      if (!_wizardLogin) {
-        _wizardLogin = document.getElementById('wizard-login');
+      if (initialized) {
```
Comment (crdlc) [inline app/js/screens/wizard/authenticate.js]: we could check if _wizardPanel  exists avoiding a new variable thought

File: app/js/screens/wizard/authenticate.js [hunk 3]
```diff
@@ -24,22 +25,24 @@
...
+      _fxaButton = document.getElementById('authenticate-fxa-button');
+      _mobileIdButton = document.getElementById('authenticate-msisdn-button');
+      _wizardLogin = document.getElementById('wizard-login');
+      _wizardPanel = document.getElementById('wizard-panel');
+      _gotItButton = document.getElementById('got-it-button');
 
       _fxaButton.addEventListener('click', _onFxaButtonClick);
       _mobileIdButton.addEventListener('click', _onMobileIdButtonClick);
+      _gotItButton.addEventListener('click', function() {
+        Navigation.to('calllog-panel', 'left');
```
Comment (crdlc) [inline app/js/screens/wizard/authenticate.js]: are the animations at the same time or should we do them consecutively?

Comment (borjasalguero) [inline app/js/screens/wizard/authenticate.js]: Changing to do it as a sequence.

File: app/js/screens/wizard/authenticate.js (hunk 2 above)
Comment (borjasalguero) [inline app/js/screens/wizard/authenticate.js]: Done

File: app/js/screens/settings.js (hunk 1 above)
Comment (borjasalguero) [inline app/js/screens/settings.js]: +1. Changing

**Pull request merged**